src/form/CustomRadioSelect.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import styles from "./CustomRadioSelect.module.css";
+
+interface Option {
+  value: string,
+  component: JSX.Element,
+  selectedComponent: JSX.Element,
+}
+
+interface Props {
+  title: string | JSX.Element,
+  name: string,
+  value: string,
+  valueChanged: (event: any) => void,
+  readonly?: boolean
+  options: Option[]
+};
+
+export const CustomRadioSelect: React.FC<Props> = (props) => {
```

Comment:
This is a nice component to have. I think we can also reuse it for the access modifier.